utils/golang/utils.go

```diff
@@ -83,8 +84,8 @@ func GetModuleName(projectDir string) (string, error) {
 	return path, nil
 }
 
-func GetDependenciesList(projectDir string) (map[string]bool, error) {
-	deps, err := utils.GetDependenciesList(projectDir, log.Logger)
+func GetDependenciesList(projectDir string, errorFunc utils.HandleErrorFunc) (map[string]bool, error) {
```

Comment:
This function seems to be unused, so let's remove it.
The same goes for the below `GetDependenciesGraph` function. Let'e remove it as well.

Reply:
It's in use in the `security` repo

Reply:
If those functions are used by the jfrog-cli-security module, let's move them to that module.

Reply:
Done